src/containers/ConversionForm/index.tsx

```diff
@@ -237,9 +237,11 @@ const ConversionForm = ({
                     )}
                 </div>
                 <Divider className={styles.divider} orientation="right" />
-                <Button ghost onClick={cancelConversion}>Cancel</Button>
+                <Button className="secondary-button" onClick={cancelConversion}>
+                    Cancel
+                </Button>
                 <Button
-                    type="primary"
+                    className="primary-button"
```

Comment:
this seems like we should have a constant for it, I'm surprised we don't already. I wouldn't do it in this PR since this PR is just trying to get it to be the same as main, but maybe make an issue for it